macOS/DuckDuckGoVPN/VPNUIActionHandler.swift

```diff
@@ -55,4 +55,8 @@ final class VPNUIActionHandler: VPNUIActionHandling {
     public func showPrivacyPro() async {
         try? await appLauncher.launchApp(withCommand: VPNAppLaunchCommand.showPrivacyPro)
     }
+
+    public func willStopVPN() async -> Bool {
+        true
```

Comment:
When stopping the VPN from the menu app, the user won't be shown anything and the VPN will just turn off.